commit 31042d6eb1d03b4e54662f51ef3366f9346617a0
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-04-26T10:42:46+02:00 (6 months ago)
    
    TRITON-1299 need logarchiver-agent service to run along side hermes-actor

diff --git a/deps/hermes b/deps/hermes
index f607e55..1cf50fa 160000
--- a/deps/hermes
+++ b/deps/hermes
@@ -1 +1 @@
-Subproject commit f607e55d1902dca6a3e97b1d7d72071a3be0ec93
+Subproject commit 1cf50fa80a674a5a427df733ac58eb162b7aadba
